### DISCUSSION TITLE
Single instance not appearing in "friend" tab in Lan Game

### DISCUSSION THREAD

**Discussion by logicwar** [itzg/docker-minecraft-bedrock-server#175] [General]
Running the container on a QNAP NAS (Container Station), I cannot get the server to show up in the "friend" tab as a lan game. (Same behaviour if I run the container in a linux VM on a PC).

The server is up and running properly and I can connect to it without a problem via the "Servers" tab if I define the server details. While this is good enough for a mobile, it needs to shop up under the "friend" tab for a PS4 console (the server tab doesn’t allow for adding manually a server).

From what I can tell, when running on QNAP or VM container, the server is not multicasting on the network.

If I run the windows version directly on a PC, the server shows properly in the “friend” tab and I can see it multicasting.

I saw the different discussion regarding “multi-instances” on same host, but here I am only running a single instance.

Any idea ?

**Comment by itzg** [comment]
As you mentioned, the bedrock addition makes use of UDP multicasting for server discovery. When it works, it's great, but there many times it doesn't work due to any number of factors. My only advice is to dry debugging the multicast process by running tcpdump on the host (**not** in the container), such as

```
sudo tcpdump -i any -vvv udp port 19132
```

**Comment by itzg** [comment]
I am going to convert this to a discussion since there is not anything more the image can do and we'll see if others have advice for this scenario.

**Reply by logicwar** [reply]
Thank you
